.changelog/unreleased/api-breaking/provider/2171-permissionless-ICS.md.md

```diff
@@ -0,0 +1,36 @@
+
+- Deprecate the `chain-id` parameter in favour of `consumer-id` for all transactions(TXs) and queries targeting a unique consumer chain. Below is a list highlighting the changes in the CLI commands. All commands assume the prefix `interchain-security-pd tx|q provider`.
+  - **TXs:**
+    | Command                                 | Message                             |
```

Comment:
Do not use a table as it's formatted weird in Markdown. Just use bullet point lists, e.g., 

- `opt-in [consumer-id] [consumer-pubkey]` -- submit a [MsgOptIn](https://github.com/cosmos/interchain-security/blob/feat/permissionless/proto/interchain_security/ccv/provider/v1/tx.proto#L256)

Reply:
Updated.